app/code/AstoundDRudenko/Badge/view/frontend/templates/product/view/info/badges.phtml

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Product info badges template
+ *
+ * @var $block AstoundDRudenko\Badge\Block\Product\View\Info\Badges
+ */
+?>
+
+<?php if ($block->productBadgesEnabled() && $badges = $block->getProductBadges()): ?>
```

Comment:
please never use variable declaration for if-statements.
that should be something like:
```
<?php if ($block->productBadgesEnabled()): ?>
   <?php $badges = $block->getProductBadges();?>
   <?php if ($badges): ?>
```